src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -116,6 +120,9 @@ export type Fiber = {
   // The state used to create the output
   memoizedState: any,
 
+  // Bitmask that tracks properties of a subtree
+  contextTag: TypeOfContext,
```

Comment:
I'll add a comment explaining what it does and a TODO to figure out a better name 👍